openapi.yaml

```diff
@@ -3554,9 +3554,13 @@ paths:
                   example: 234
                 overwrite:
                   type: boolean
-                  description: >
+                  description: |
                     If True, deletes all Disks and Configs on the target Linode
                     before restoring.
+
+                    If False, and the Disk image size is less than the available
```

Comment:
shouldn't this be if the disk image size is MORE than the available space on the Linode?